Sem11/intersection.py

```diff
@@ -0,0 +1,23 @@
+import sympy.polys.polytools
+from sympy import *
+import variables
+
+
+def get_poly_sum():
+    f1 = variables.f1
+    f2 = variables.f2
+    poly1 = sympy.polys.polytools.poly_from_expr(str(f1))[0]
+    poly2 = sympy.polys.polytools.poly_from_expr(str(f2))[0]
+    return (poly1 + poly2).as_expr()
```

Comment:
Вот тут у вас ошибка. Если вы хотите найти точки пересечения, то функции нужно приравнять друг к другу. Это значит, что результирующая функция будет функцией их разности. К тому же их разность, как и сумму, можно идти куда проще, чем так, как сделали вы:
```suggestion
    return f1 - f2  # или f2 - f1
```

Reply:
> Это значит, что результирующая функция будет функцией их разности.

точно, это же ясно как день.. не догадался сам(

Reply:
переделаю сегодня..

Reply:
> К тому же их разность, как и сумму, можно найти куда проще, чем так, как сделали вы:

а как? подскажите направление

Reply:
Так я  уже подсказал в Suggested change